public/admin/main.css

```diff
@@ -207,8 +207,8 @@ body.state-playing-episode {
 
 .cue-controls input[type=range].cue-controls__cue-controls-go {
   -webkit-appearance: none;
-  width: 100%;
-  margin: 1rem 0; }
+  width: 90%;
```

Comment:
Were these changes you made deliberately? If not then I will revert them — the admin site and the project site shouldn't have any effect on each other...

Reply:
The changes to main.css were made when I ran "yarn run dev." I hadn't gone in to the file and changed anything. I'll attach an image from the terminal.

<img width="565" alt="Screen Shot 2019-10-03 at 6 49 33 PM" src="https://user-images.githubusercontent.com/9770973/66169664-2ddf0800-e60f-11e9-8aef-319a7a3c32e5.png">